homeassistant/components/vicare/strings.json

```diff
@@ -234,28 +234,40 @@
         "name": "DHW gas consumption last seven days"
       },
       "energy_summary_consumption_heating_currentday": {
-        "name": "Heating energy consumption today"
+        "name": "Electrical energy consumption for heating today"
       },
       "energy_summary_consumption_heating_currentmonth": {
-        "name": "Heating energy consumption this month"
+        "name": "Electrical energy consumption for heating this month"
       },
       "energy_summary_consumption_heating_currentyear": {
-        "name": "Heating energy consumption this year"
+        "name": "Electrical energy consumption for heating this year"
       },
       "energy_summary_consumption_heating_lastsevendays": {
-        "name": "Heating energy consumption last seven days"
+        "name": "Electrical energy consumption for heating last seven days"
       },
       "energy_dhw_summary_consumption_heating_currentday": {
-        "name": "DHW energy consumption today"
+        "name": "Electrical energy consumption for DHW today"
       },
       "energy_dhw_summary_consumption_heating_currentmonth": {
-        "name": "DHW energy consumption this month"
+        "name": "Electrical energy consumption for DHW this month"
       },
       "energy_dhw_summary_consumption_heating_currentyear": {
-        "name": "DHW energy consumption this year"
+        "name": "Electrical energy consumption for DHW this year"
```

Comment:
Okay I saw that there was a whole discussion (I did not read, sorry for that), but let me ask a quick question, would it make sense to keep the start of the name contain the type? Because if I imagine a device with these entities, on the device page everything looks like `Electrical energy consum...`. And if I translate it to Dutch `Energie consumptie ...`. (I am not sure how wide it will show, but with this long name, I can see it being cut off, that's my point).

Would it make sense to have `Heating electrical energy consumption today`? (under the assumption the reason this PR exists is because of the added emphasis on `eletrical`).

(I think usually I wouldn't make this big of a deal of this, but in this case it can be 12(?) entities, so that would most likely lead to people renaming the things).

(Also, added question, on what level is this information collected? Would it make sense to only show the heating usage on the heating device and DHW on the DHW device?)